jwt/api.py

```diff
@@ -233,6 +254,19 @@ def _validate_claims(self, payload, verify_expiration=True, leeway=0,
             if payload.get('iss') != issuer:
                 raise InvalidIssuerError('Invalid issuer')
 
+    def _merge_options(self, override_options=None):
+        if not override_options:
+            override_options = {}
+
+        try:
+            merged_options = dict()
+            for k, v in self.options.items():
+                merged_options[k] = override_options.get(k, v)
```

Comment:
This looks very similar to my other comment about using `dict.update()` (see above)

This code looks very similar. You could probably break it out into a function and reuse it in both spots.